share/picongpu/examples/LaserWakefield/etc/picongpu/4_saxs.cfg

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2020 Axel Huebl, Rene Widera, Felix Schmitt,
+#              Pawel Ordyna
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      docs/TBG_macros.cfg
+##
```

Comment:
I am fine with having a SAX case in the LWFA case since this is the common starting point, but @pordyna and @HighIander does it make sense from a physics point of view?

Reply:
I missed this one. It can be deleted.